I can instantiate a 1.0 template when the 1.0 runtime is not installed on my machine.

Split out of https://github.com/GoogleCloudPlatform/google-cloud-visualstudio/issues/847

And when I try to publish, I get this error:

```
Publishing GcpProject2 to Compute Engine.
dotnet publish -o "C:\Users\Jeffrey Rennie\AppData\Local\Temp\jpolqxei.lnu" -c Release
Microsoft (R) Build Engine version 15.4.8.50001 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  GcpProject2 -> C:\Users\Jeffrey Rennie\source\repos\GcpProject2\GcpProject2\bin\Release\netcoreapp2.0\GcpProject2.dll
  It was not possible to find any compatible framework version
  
  The specified framework 'Microsoft.NETCore.App', version '1.0.0' was not found.
  
    - Check application dependencies and target a framework version installed at:
  
        \
  
    - Alternatively, install the framework version '1.0.0'.
  
C:\Users\Jeffrey Rennie\source\repos\GcpProject2\GcpProject2\GcpProject2.csproj(24,5): error MSB3073: The command "dotnet bundle" exited with code -2147450749.
Failed to deploy project GcpProject2 to App Engine Flex.
```

--- a/GoogleCloudExtension/GoogleCloudExtension.GCloud/Models/GkeList.cs
+++ b/GoogleCloudExtension/GoogleCloudExtension.GCloud/Models/GkeList.cs
@@ -18,7 +18,7 @@ using System.Collections.Generic;
 namespace GoogleCloudExtension.GCloud.Models
 {
     /// <summary>
-    /// Common class for all of the lists returned from Kuberentes.
+    /// Common class for all of the lists returned from Kubernetes.
     /// </summary>
     /// <typeparam name="T"></typeparam>
     public class GkeList<T>

--- a/GoogleCloudExtension/ProjectTemplates/GcpProjectTemplate/Core/MVC/1.0.csproj
+++ b/GoogleCloudExtension/ProjectTemplates/GcpProjectTemplate/Core/MVC/1.0.csproj
@@ -28,7 +28,7 @@
   </Target>
 
   <ItemGroup>
-    <DotNetCliToolReference Include="BundlerMinifier.Core" Version="2.2.301" />
+    <DotNetCliToolReference Include="BundlerMinifier.Core" Version="2.6.362" />
   </ItemGroup>
 
 </Project>

--- a/GoogleCloudExtension/ProjectTemplates/GcpProjectTemplate/Core/MVC/1.1.csproj
+++ b/GoogleCloudExtension/ProjectTemplates/GcpProjectTemplate/Core/MVC/1.1.csproj
@@ -28,7 +28,7 @@
   </Target>
 
   <ItemGroup>
-    <DotNetCliToolReference Include="BundlerMinifier.Core" Version="2.2.301" />
+    <DotNetCliToolReference Include="BundlerMinifier.Core" Version="2.6.362" />
   </ItemGroup>
 
 </Project>

--- a/GoogleCloudExtension/ProjectTemplates/GcpProjectTemplate/Core/MVC/2.0.csproj
+++ b/GoogleCloudExtension/ProjectTemplates/GcpProjectTemplate/Core/MVC/2.0.csproj
@@ -25,7 +25,7 @@
   </Target>
 
   <ItemGroup>
-    <DotNetCliToolReference Include="BundlerMinifier.Core" Version="2.2.301" />
+    <DotNetCliToolReference Include="BundlerMinifier.Core" Version="2.6.362" />
   </ItemGroup>
 
 </Project>

--- a/GoogleCloudExtension/ProjectTemplates/GcpProjectTemplate/Core/MVC/project.json
+++ b/GoogleCloudExtension/ProjectTemplates/GcpProjectTemplate/Core/MVC/project.json
@@ -22,7 +22,7 @@
   },
 
   "tools": {
-    "BundlerMinifier.Core": "2.0.238",
+    "BundlerMinifier.Core": "2.6.362",
     "Microsoft.AspNetCore.Razor.Tools": "1.0.0-preview4-final",
     "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-preview2-final"
   },

